browser/src/layer/vector/SVGGroup.js

```diff
@@ -76,6 +76,11 @@ L.SVGGroup = L.Layer.extend({
 		this._update();
 	},
 
+	isCalcRTL: function () {
```

Comment:
This exists in other file(s) too. Perhaps it's better to make this part of `docLayer` and replace it with a single `this._map._docLayer.isCalcRTL()` call? Sure it's not great to have application-specific helpers in docLayer (calc, in this case), but copy pasting these 4 lines is probably worse.